core/FileIntegrity.php

```diff
@@ -138,7 +138,7 @@ protected static function getMessagesFilesFoundButNotExpected($messages)
 
             $files = array();
             foreach ($filesFoundButNotExpected as $fileFoundNotExpected) {
-                $files[] = '"' . htmlspecialchars(realpath($fileFoundNotExpected)) . '"';
+                $files[] = '"' . htmlspecialchars($fileFoundNotExpected) . '"';
```

Comment:
```suggestion
                $files[] = '"' . htmlspecialchars(realpath(dirname($fileFoundNotExpected)). DIRECTORY_SEPARATOR . basename($fileFoundNotExpected)) . '"';

```

Reply:
Thanks for the suggestions, and I'm sorry I somehow missed your earlier comment + suggestion! Thanks for taking the time to evaluate and consider a hopefully good balance between past behavior and avoiding the bug. I'll update the PR and as you say, see how it goes!